index.ios.js

```diff
@@ -1,4 +1,4 @@
 import { AppRegistry } from 'react-native';
-import { App } from './main';
+import { App } from './build';
```

Comment:
Yes, as I renamed `src/main.tsx` to `src/index.ts` the resulting `build/main.js` will now be `build/index.js` and we can require it as `'./build'`